CMakeLists.txt

```diff
@@ -61,6 +61,8 @@ set(COG_HOME_URI "" CACHE STRING "Default home URI")
 set(COG_MODULEDIR "${CMAKE_INSTALL_PREFIX}/lib/cog/modules"
     CACHE STRING "Default search path for loadable modules")
 
+set(COG_DEVICE_SCALING 1)
```

Comment:
I did that to avoid dependencies (in the includes) with  wpebackend-fdo-1.0 (`WPE_CHECK_VERSION`) for the platform. In other case `wpebackend-fdo-1.0` has to be added to the required modules `pkg_check_modules` for the platform-wl library as well.

Reply:
Not really, it's possible to check whether the macro is defined before trying to use it:

```c
#if defined(WPE_FDO_CHECK_VERSION) && WPE_FDO_CHECK_VERSION(1, 3, 0)
  /* ... */
#endif 
```

Reply:
The point is this not only checks the `WPE_FDO_CHECK_VERSION` but also the version number. Anyway I'm fine with using `WAYLAND_VERSION` instance of `WPE_FDO_CHECK_VERSION`  as you suggested in the comment below.